### PR TITLE
[Flutter] Update Unimodules Flutter Adapter

### PR DESCRIPTION
**Pull request by FiberJW**
# Why

After the EX -> UM rename (EXCore -> UMCore, for example), the Flutter adapter needed to be updated to keep up with those changes.

# How

I updated the podspec and gradle files for iOS and Android respectively to match the new unimodules framework package names.

# Test Plan

Make sure that you have Flutter installed,`cd` into `packages/expo-flutter-adapter/example`, `yarn`, and run `flutter run` with a connected physical device to test that the updated adapter works with the currently existing permissions and sensors PoC modules.

Once approved/merged, I'll publish this update to the Flutter/Dart package registry, pub.dev.

Thanks!

### CODE REVIEW

File: packages/expo-flutter-adapter/example/ios/Flutter/flutter_export_environment.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+# This is a generated file; do not edit or check into version control.
```
Comment (sjchmiela) [inline packages/expo-flutter-adapter/example/ios/Flutter/flutter_export_environment.sh]: > or check into version control

🙂

File: packages/expo-flutter-adapter/example/ios/Podfile.lock [hunk 2]
```diff
@@ -44,22 +38,26 @@ EXTERNAL SOURCES:
...
+  Flutter: 0e3d915762c693b495b44d77113d4970485de6ec
+  UMCore: 733094f43f7244c60ce1f0592d00013ed68fa52c
+  UMPermissionsInterface: 938d010c74c43fcefc9bb990633a7c5a1631267e
+  UMSensorsInterface: 0ed023ce9b96f2ca6fada7bda05b7760da60b293
 
-PODFILE CHECKSUM: a1d5878d34906d9539cefd078104709223ffa495
+PODFILE CHECKSUM: 4cba2060905e5e3d99dfdc32ca70aec0ec441dbb
 
-COCOAPODS: 1.5.3
+COCOAPODS: 1.8.0.beta.2
```
Comment (sjchmiela) [inline packages/expo-flutter-adapter/example/ios/Podfile.lock]: How come you're not using `expo/expo`'s (Nix's) `pod`? 🤔

File: packages/expo-flutter-adapter/example/package.json [hunk 3]
```diff
@@ -6,8 +6,8 @@
   "dependencies": {
     "@unimodules/core": "file:../../@unimodules/core",
     "expo-permissions": "file:../../expo-permissions",
-    "expo-permissions-interface": "file:../../expo-permissions-interface",
+    "unimodules-permissions-interface": "file:../../unimodules-permissions-interface",
```
Comment (sjchmiela) [inline packages/expo-flutter-adapter/example/package.json]: Couldn't we use Yarn workspaces for this?

File: packages/expo-flutter-adapter/ios/Classes/ExpoFlutterAdapterPlugin.m [hunk 4]
```diff
@@ -106,41 +107,41 @@ - (void)applicationDidEnterBackground:(nonnull UIApplication *)application {
...
-    NSLog(@"%@", message);
-    return nil;
-}
-
-extern UIApplication *EXSharedApplication() {
-    return nil;
-}
+#pragma mark UMCore Logging Blocks
+
+//extern void UMLogInfo(NSString *format, ...) {
```
Comment (sjchmiela) [inline packages/expo-flutter-adapter/ios/Classes/ExpoFlutterAdapterPlugin.m]: True, this should be removable! The definition now lives in `UMCore`. 

https://github.com/expo/expo/blob/master/packages/%40unimodules/core/ios/UMCore/Services/UMLogManager.m#L50

**Review by sjchmiela — CHANGES_REQUESTED** [review]
Nice work! I hope that wasn't too difficult to upgrade the adapter to new unimodules. 💪

File: packages/expo-flutter-adapter/example/ios/Podfile.lock (hunk 2 above)
Comment (FiberJW) [reply packages/expo-flutter-adapter/example/ios/Podfile.lock]: Not sure actually. I updated cocoapods on my machine globally which is why it's on the 1.8.0.beta.2, but is there a way for me to use Expo's version while inside the expo/expo repo?

File: packages/expo-flutter-adapter/example/ios/Flutter/flutter_export_environment.sh (hunk 1 above)
Comment (FiberJW) [reply packages/expo-flutter-adapter/example/ios/Flutter/flutter_export_environment.sh]: Ah haha this wasn't in the gitignore when I first generated the plugin so this must be something new Flutter added. Will remove and add to the gitignore now!

**Pull request merged**